crates/net/network/src/transactions/validation.rs

```diff
@@ -0,0 +1,555 @@
+
+                return ValidationOutcome::ReportPeer
+            }
+        };
+
+        //
+        // 2. checks if tx's encoded length is within limits for this network
+        //
+        // transactions that are filtered out here, may not be spam, rather from benevolent peers
+        // that are unknowingly sending announcements with invalid data.
+        //
+        if let Some(strict_min_encoded_tx_length) = self.strict_min_encoded_tx_length(tx_type) {
+            if size < strict_min_encoded_tx_length {
+                debug!(target: "net::eth-wire",
+                    ty=ty,
+                    size=size,
+                    hash=%hash,
+                    strict_min_encoded_tx_length=strict_min_encoded_tx_length,
+                    network=%Self,
+                    "invalid tx size in eth68 announcement"
+                );
+
+                return ValidationOutcome::Ignore
+            }
+        }
+        if let Some(reasonable_min_encoded_tx_length) = self.min_encoded_tx_length(tx_type) {
+            if size < reasonable_min_encoded_tx_length {
+                debug!(target: "net::eth-wire",
+                    ty=ty,
+                    size=size,
+                    hash=%hash,
+                    reasonable_min_encoded_tx_length=reasonable_min_encoded_tx_length,
+                    strict_min_encoded_tx_length=self.strict_min_encoded_tx_length(tx_type),
+                    network=%Self,
+                    "tx size in eth68 announcement, is unreasonably small"
+                );
+
+                // just log a tx which is smaller than the reasonable min encoded tx length, don't
+                // filter it out
+            }
+        }
+        // this network has no strict max encoded tx length for any tx type
+        if let Some(reasonable_max_encoded_tx_length) = self.max_encoded_tx_length(tx_type) {
+            if size > reasonable_max_encoded_tx_length {
+                debug!(target: "net::eth-wire",
+                    ty=ty,
+                    size=size,
+                    hash=%hash,
+                    reasonable_max_encoded_tx_length=reasonable_max_encoded_tx_length,
+                    strict_max_encoded_tx_length=self.strict_max_encoded_tx_length(tx_type),
+                    network=%Self,
+                    "tx size in eth68 announcement, is unreasonably large"
+                );
+
+                // just log a tx which is bigger than the reasonable max encoded tx length, don't
+                // filter it out
+            }
+        }
+
+        ValidationOutcome::Fetch
+    }
+
+    fn max_encoded_tx_length(&self, ty: TxType) -> Option<usize> {
+        // the biggest transaction so far is a blob transaction, which is currently max 2^17,
+        // encoded length, nonetheless, the blob tx may become bigger in the future.
+        match ty {
+            TxType::Legacy | TxType::EIP2930 | TxType::EIP1559 => {
+                Some(MAX_ENCODED_PRE_BLOB_TX_LENGTH)
```

Comment:
this is too low, because this is the size of a single blob

I'd feel more at ease if we use the max eth packet size for now to unblock this pr and then figure out an appropriate limit later

Reply:
it's not the strict limit anyhow, it's just for helpful debug logs.